release/air_examples/dreambooth/dreambooth_run.sh

```diff
@@ -20,46 +18,72 @@ export IMAGES_REG_DIR="$DATA_PREFIX/images-reg"
 export IMAGES_OWN_DIR="$DATA_PREFIX/images-own"
 export IMAGES_NEW_DIR="$DATA_PREFIX/images-new"
 
-export CLASS_NAME="lego car"
-
 mkdir -p $ORIG_MODEL_DIR $TUNED_MODEL_DIR $IMAGES_REG_DIR $IMAGES_OWN_DIR $IMAGES_NEW_DIR
 
-# Copy own images into IMAGES_OWN_DIR
-cp -rf ./images/unqtkn/*.jpg "$IMAGES_OWN_DIR/"
+# Unique token to identify our subject (e.g., a random dog vs. our unqtkn dog)
+export UNIQUE_TOKEN="unqtkn"
 
 # Step 1
+# Only uncomment one of the following:
+
+# Option 1: Use the dog dataset ---------
+export CLASS_NAME="dog"
```

Comment:
Nice! I also agree to use dog as the target class, since it aligns with the experiment setting in the original paper.